src/modules/components/proposal/proposalActions/proposalActionsContainer/proposalActionsContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import { cloneElement, isValidElement, useState, type ReactElement } from 'react';
+import { Accordion, Button, Card, Heading, type IAccordionContainerBaseProps } from '../../../../../core';
+import { useOdsModulesContext } from '../../../odsModulesProvider';
+import { type IProposalActionsActionProps } from '../proposalActionsAction';
+
+export interface IProposalActionsContainerProps extends Omit<IAccordionContainerBaseProps<true>, 'isMulti'> {
```

Comment:
Do we need to expose any accordion-container specific prop? Probably no for now, I'd simply extend the `ComponentProps<'div'>` interface and pass all other-props to the card component (make sure to avoid overriding the `className` component)